openmdao/docs/openmdao_book/features/core_features/working_with_groups/src_indices.ipynb

```diff
@@ -35,17 +35,16 @@
     "if your output is an array and you only want to connect part of it to your\n",
     "input?  \n",
     "\n",
-    "If you connect variables via promotion, you must set `src_indices` either\n",
-    "when you call `add_input` to add your input variable to its component or \n",
-    "when you promote the input with the [promotes](group-promotion) method.  Another\n",
+    "If you connect variables via promotion, you must set `src_indices` when you promote the input with the [promotes](group-promotion) method,   Another\n",
     "argument, `flat_src_indices` is a boolean that determines whether the entries\n",
     "of the `src_indices` array are interpreted as indices into the flattened source\n",
     "or as tuples or lists of indices into the unflattened source.  The default\n",
     "of `flat_src_indices=False` assumes indices map into an unflattened source.\n",
     "\n",
-    "Usage\n",
     "\n",
-    "1: Connect an independent array variable to two different components where each component gets part of the array.  "
+    "#### Basic Example\n",
```

Comment:
This header needs to have only two hash marks, because the previous header is one hash mark.  That's the meaning of this warning, which is causing the CI failure:
```/home/runner/work/OpenMDAO/OpenMDAO/openmdao/docs/openmdao_book/features/core_features/working_with_groups/src_indices.ipynb:20015: WARNING: Non-consecutive header level increase; 1 to 4 [myst.header]```